### PR TITLE
add group relationship for a team

### PR DESCRIPTION
**Pull request by psaffaie**
adding documentation how a group and a team are related

### CODE REVIEW

**Review by FaithOmbongi — APPROVED** [review]
Thank you @psaffaie for closing this doc gap.

File: api-reference/beta/resources/group.md [hunk 1]
```diff
@@ -204,6 +204,8 @@ This resource supports:
...
 | sites                    | [site](site.md) collection                                            | The list of SharePoint sites in this group. Access the default site with /sites/root.                                                                                                                                                                                                                                                                                                                                                                                       |
+| team                    | [channel](channel.md) collection                                            | Every team is associated with a [Microsoft 365 group](../resources/group.md). The group has the same ID as the team - for example, `/groups/{id}/team` is the same as `/teams/{id}`.For more information about working with groups and members in teams, see [Use the Microsoft Graph REST API to work with Microsoft Teams](teams-api-overview.md). |
```
Comment (Lauragra) [inline api-reference/beta/resources/group.md:207]: Please see my suggested revision for the description on the other PR related to this one. We will need a changelog entry for both the beta and the v1.0 update.

Comment (FaithOmbongi) [reply api-reference/beta/resources/group.md]: Hi @Lauragra, this is a change to resolve the out-of-sync schema/docs issues. We're avoiding new changelogs for these properties and relationships that may have existed in the schema for a long time but lacked related docs.

**Comment by psaffaie** [comment]
Closing since we made updates to the team relationship on both beta and v1 versions in PR https://github.com/microsoftgraph/microsoft-graph-docs/pull/16390

**Pull request closed without merging**